pkg/build/metrics/prometheus/metrics.go

```diff
@@ -59,11 +67,21 @@ type buildCollector struct {
 func IntializeMetricsCollector(buildLister buildlister.BuildLister) {
 	if !bc.IsCreated() {
 		bc.lister = buildLister
-		legacyregistry.MustRegister(&bc)
+		legacyregistry.MustRegister(&bc, triggerClearedCounter)
 	}
 	klog.V(4).Info("build metrics registered with prometheus")
 }
 
+// MarkImageChangeTriggerCleared sets the openshift_build_image_change_trigger_cleared gague value.
+// A value of 1 indicates the LastTriggeredImageID field on a BuildConfig has been reset to empty.
+func MarkImageChangeTriggerCleared(namespace, name string, cleared bool) {
+	value := float64(0)
+	if cleared {
+		value = float64(1)
+	}
+	triggerClearedCounter.WithLabelValues(namespace, name).Set(value)
+}
+
```

Comment:
Here we set the metric value to 0 or 1 - eventually every `BuildConfig` in the cluster will have a value set.